pep-0001.txt

```diff
@@ -788,7 +787,8 @@ References and Footnotes
 Copyright
 =========
 
-This document has been placed in the public domain.
+This document is placed in the public domain or under the
```

Comment:
That seems like either-or, not both.

Reply:
This is straight from @VanL . So since neither of us are a lawyers and Van is I'm going to say you're wrong and Van's right. 😉